scenarios/02_Fighting_for_Passage.cfg

```diff
@@ -324,19 +335,64 @@
                 description=_"Death of Reme Carrenemoe"
                 condition=lose
             [/objective]
+            [gold_carryover]
+                bonus=yes
+                # see note in end_level tag below on these percentages:
+                carryover_percentage={ON_DIFFICULTY 80 70 60}
+            [/gold_carryover]
+            {TURNS_RUN_OUT}
+            {OBJECTIVE_NOTES _"This scenario is designed for grinding gold and EXP. Your income will increase if you use Gawen for fighting instead of recruiting, so be sure to move him off your keep and into the action!"}
```

Comment:
Yeah I don't think this is the way to go. I'd rather you balance it another way and objective notes aren't for strategy hints.

Reply:
I disagree; lots of other campaigns use objective notes for strategy hints and I find it very helpful when they do so. And varying gold carryover by difficulty is a common thing I do with my campaigns and I think it really helps keep each difficulty level fresh.

Reply:
Also the note isn't just a strategy hint, it's the only way the income-changing mechanic is described for players. Without it, players might get confused as to how their income works.

Reply:
That's bad design on the part of the campaign imo. Almost no other game I'm aware of gives you such direct advice on how to play except for tutorial style campaigns. One of the things about game design is that of course there's an intention on how a scenario is supposed to be played, but players will almost always find other ways to play. So yeah, I'm okay if you want to make it a note on easy, but please don't do it for other modes.

Varying gold carryover is fine, but you should be consistent about it between scenarios. Doesn't make sense to only vary it for one scenario.

Reply:
ok, I varied the note by difficulty in 6f04548. I didn't remove it entirely on the harder difficulties, though, but instead just made it less of a hint, as I still think it's important to explain to the player how the income mechanic works.
As for the gold carryover, I gave in and just removed the variance on it here because I didn't want to go and change the rest of the scenarios, too. Note that some of them are already inconsistent and grant a full 100% carryover. The only reason I felt like this one in particular should vary is because it's the only one I've received feedback about carryover leading to the following scenario having too much gold. Maybe @Asheviere can suggest something else if varying the carryover percentage isn't going to fly.